components/layout/Footer.tsx

```diff
@@ -2,45 +2,87 @@ import styled from "@emotion/styled";
 import map from "lodash.map";
 import { GREY_2, HEADLINE_TEXT, PARAGRAPH_TEXT } from "../colors";
 import { POPPINS, ROBOTO } from "../typography";
-import { breakUp } from "../breakpoints";
+import { breakUp, breakDown } from "../breakpoints";
+import logo from "../../assets/images/dwolla-developers-logo.png";
 
 const Container = styled.div`
   border-top: 1px solid ${GREY_2};
   display: flex;
   flex-wrap: wrap;
 
-  @media (${breakUp("lg")}) {
+  @media (${breakUp("md")}) {
     padding: 20px;
   }
 `;
 
-const Group = styled.div`
+const LogoContainer = styled.div`
+  width: 35%;
+  padding: 20px;
+
+  @media (${breakDown("sm")}) {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+`;
+
+const Logo = styled.img`
+  width: 173px;
+`;
+
+const Copyright = styled.div`
+  font-family: ${ROBOTO};
+  font-size: 12px;
+  line-height: 21px;
+  color: ${PARAGRAPH_TEXT};
+  margin-top: 14px;
+`;
+
+const LinkContainer = styled.div`
```

Comment:
Could we do something here to allow the footer links to stick more to the left side of the screen (closer to the logo) than the right side? This looks great on a regular desktop and smaller screens, but when the browser stretched out, the links stick to the right leaving some white space between the logo and the link.

Reply:
The main thing I was trying to avoid was setting an arbitrary max-width, but you're right that it does look off. My latest commit should address it by making the max-width equal to the width of the content area when the OnThisPage component is present. Thanks for having me take another look!

Reply:
Thanks, Connor. That looks neat! 👍